src/communication/ipc_ws.js

```diff
@@ -69,13 +75,13 @@ class IpcWs extends Communication {
                 return;
             }
 
-            for (let i = 0; i < ele.length; i++) {
-                if (!ele[i].id) {
-                    this.subscribeMethod && this.subscribeMethod(ele[i]);
+            for (const x of ele) {
```

Comment:
Not immediately clear what `x` or `ele` are. I prefer more descriptive variable names.

Reply:
I agreed. This PR is a quick fix for webscoket failed reconnection. I should leave those changes for another.